CMakeLists.txt

```diff
@@ -246,7 +246,9 @@ if(CORENRN_ENABLE_LEGACY_UNITS)
 else()
   set(CORENRN_USE_LEGACY_UNITS 0)
 endif()
+# Propagate Legacy Units flag to backends.
 set(MOD2C_ENABLE_LEGACY_UNITS ${CORENRN_ENABLE_LEGACY_UNITS} CACHE BOOL "" FORCE)
+set(NMODL_ENABLE_LEGACY_UNITS ${CORENRN_ENABLE_LEGACY_UNITS} CACHE BOOL "" FORCE)
 
```

Comment:
We can use option instead of `set` because there are option in sub project.
Any wish @pramodk ?

Reply:
if there will be `options` twice, would that work same way? I don't have any preference.